contact.html

```diff
@@ -120,7 +127,7 @@
             description de votre intérêt de partnenariat avec nous ci-dessous.</label>
           <textarea id="partner-description" class="contact-input" name="message"></textarea>
         </p>
-       
+        <div data-netlify-recaptcha></div>

```

Comment:
De ce que je vois, le captcha a juste été ajouté pour le formulaire des partenaires mais pas pour les nouveaux membres.

Formulaire des membres:
![screen shot 2018-11-09 at 10 05 29 pm](https://user-images.githubusercontent.com/1891861/48296926-a83cf180-e46b-11e8-9f7e-626a3adc602e.png)

Formulaire des partenaires:
![screen shot 2018-11-09 at 10 05 46 pm](https://user-images.githubusercontent.com/1891861/48296930-ae32d280-e46b-11e8-9dae-00a7e43dcb19.png)

Reply:
Uhmmm that one is weird parce que le div est dans les deux forms.